File: Урок 6/ex2.cpp
/* Задача 2: Скидка в магазине (Оператор ||)
Задание: Магазин дает скидку, если сегодня суббота ИЛИ сегодня воскресенье.

Создай переменную dayNumber (1 — понедельник, 7 — воскресенье).

Если число равно 6 или 7, выведи: "You get a discount!".

В остальные дни выведи: "Full price today.". */


#include <iostream>
using namespace std;

int main() {
    int dayNumber;
    cout << "Enter day number (1-7): ";
    cin >> dayNumber;

    if (dayNumber == 6 || dayNumber == 7) {
        cout << "You get a discount!" << endl;
    } else {
        cout << "Full price today." << endl;
    }
    return 0;
}

File: Python/Урок 16/ex2.py
'''
Задача 2. «Калькулятор скидки» (Return + Default)
Задание: Напишите функцию calculate_price, которая принимает:
- price (цена товара).
- discount (скидка в процентах). По умолчанию скидка должна быть 0. 
Функция должна возвращать итоговую цену.
'''




























def calculate_price(price, discount=0):
    final_price = price - (price * discount / 100)
    return final_price

# Тесты
print(calculate_price(1000, 20)) # Должно быть 800.0
print(calculate_price(500))      # Должно быть 500 (скидка 0)